MAUI/Chips/Customization.md

```diff
@@ -1806,4 +1806,8 @@ using Syncfusion.Maui.Core;
 
 N> The default value of ChipImageSize is [`18d`].
 
+## IsSelected
```

Comment:
Add this title under the selection properties

Reply:
Addressed the comments